spec/Section 6 -- Execution.md

```diff
@@ -143,6 +143,84 @@ ExecuteMutation(mutation, schema, variableValues, initialValue):
     selection set.
   * Return an unordered map containing {data} and {errors}.
```

Comment:
Also noticed that a step should be added to `{ExecuteRequest}` above to handle subscription requests.